### PR TITLE
Add ability for facilitator to highlight an idea

### PR DESCRIPTION
**Pull request by samdec11**
Closes https://github.com/stride-nyc/remote_retro/issues/157

### CODE REVIEW

File: web/static/js/components/idea.jsx [hunk 1]
```diff
@@ -8,9 +10,12 @@ const Idea = props => {
   const { idea, currentUser, retroChannel } = props
   const isFacilitator = currentUser.is_facilitator
   const isEdited = (+new Date(idea.updated_at) - +new Date(idea.inserted_at)) > 1000
-  let classes = styles.index
-  classes += idea.editing ? " ui raised segment" : ""
-  classes += idea.highlighted ? ` ${styles.highlighted}` : ""
+  const classes = classNames({
```
Comment (vanderhoop) [inline web/static/js/components/idea.jsx]: nice work here. much nicer implementation

**Comment by vanderhoop** [comment]
hey @samdec11, really nice work here. this feature has a lot of moving parts, and we should have an end to end test to protect us against regressions somewhere in the chain. add an end to end test checking that highlighting works across clients and this puppy is mergeable

**Comment by samdec11** [comment]
@vanderhoop How's that?

**Comment by vanderhoop** [comment]
Monsieur Moneybags! Mergin'

**Pull request merged**